app/models/developers/title.rb

```diff
@@ -0,0 +1,49 @@
+module Developers
+  class Title
+    private attr_reader :query
+
+    def initialize(query)
+      @query = query
+    end
+
+    def title
+      components = []
+      components << I18n.t("hire", scope:)
+      components << role_level if role_level
+      components << I18n.t("freelance", scope:) if freelance?
+      components << I18n.t("ruby_on_rails_developers", scope:)
+      components << I18n.t("in_country", country:, scope:) if country
+      components.join(" ")
+    end
+
+    private
+
+    def scope
+      "developers.title"
+    end
```

Comment:
I feel like there's a better approach to this. But trying to make anything more dynamic makes `bundle exec i18n-tasks missing en` fail because it can't find the translations.

Reply:
I've gone with manual commenting of the used keys for now. I don't love it, but it works.